bin/direxec

```diff
@@ -53,6 +53,14 @@ if extra_command:
                 + os.environ["PTVSD_ARGS"].split()
                 + extra_command
             )
+        if os.environ["DEBUGPY_ENABLE"] == "1":
```

Comment:
We don't want both ptvsd and debugpy, right?

```suggestion
        elif os.environ["DEBUGPY_ENABLE"] == "1":
```

Also please add some deprecation notice when using ptvsd instead.